Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -681,6 +681,10 @@ theorem comm_symm_tmul (m : M) (n : N) : (TensorProduct.comm R M N).symm (n ⊗
   rfl
 #align tensor_product.comm_symm_tmul TensorProduct.comm_symm_tmul
 
+lemma lift_comp_comm_eq  {f : M →ₗ[R] N →ₗ[R] P} :
+    (lift f).comp (TensorProduct.comm R N M).toLinearMap =
```

Comment:
I suspect `toLinearMap` is no longer the simp-normal form, and the preferred spelling uses a coercion instead. I suspect it needs a type situation, but
```suggestion
    lift f ∘ₗ TensorProduct.comm R N M =
```

Reply:
`simp`could made no progress on the proof of the lemma;
was it enough to assume it was in simp normal form?
Anyway I made the change.